### PR TITLE
First modification to file

### PR DESCRIPTION
**Pull request by louievandyke**
Meme comment #2

### CODE REVIEW

File: _posts/0000-01-14-louievandyke.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![puss-in-boots](https://cloud.githubusercontent.com/assets/16547949/25401020/740dd682-29c2-11e7-9906-ac5c3a8496c7.jpg)
 {: .center}
 
-Caption here
+I love training on Mondays...
```
Comment (beardofedu) [inline _posts/0000-01-14-louievandyke.md]: Maybe be more specific about what kind of training, like `I :heart: GitHub training on Mondays`.

Comment (aninda-mukherjee) [reply _posts/0000-01-14-louievandyke.md]: comments from Aninda

**Comment by aninda-mukherjee** [comment]
hello comments..

**Review by aninda-mukherjee — DISMISSED** [review]
please review my new changes.

**Review by louievandyke — COMMENTED** [review]
I added the changes

**Review by spoondla1 — APPROVED** [review]
good :ship 👍

**Comment by aninda-mukherjee** [comment]
not approved.. Changes rejected..

**Review by gyenney — APPROVED** [review]
Nice work!

**Pull request merged**